index.qmd

```diff
@@ -54,10 +54,8 @@ The diagram below depicts how some of the cloud-optimized formats discussed in t
 Notes:
 
 - Some data formats cover multiple data types, specifically:
-    - GeoJSON can be used for vector and point cloud data.
```

Comment:
I don't think there is any limitation for putting Point (X, Y, Z) in a GeoJSON https://www.rfc-editor.org/rfc/rfc7946#page-7